src/main/java/net/atos/zac/aanvraag/ProductAanvraagService.java

```diff
@@ -79,6 +86,15 @@ public void verwerkProductAanvraag(final URI productAanvraagUrl) {
         zaak.setStartdatum(object.getRecord().getStartAt());
         zaak.setBronorganisatie(BRON_ORGANISATIE);
         zaak.setVerantwoordelijkeOrganisatie(BRON_ORGANISATIE);
+        try {
+            CommunicatieKanaal communicatieKanaal = vrlClientService.findCommunicatiekanaal("E-formulier");
+            zaak.setCommunicatiekanaal(communicatieKanaal.getUrl());
```

Comment:
Hier even een null check uitvoeren. Na de vorige aanpassingen.

Reply:
Opgelost in meest recente commit